src/models/customs-declarations.js

```diff
@@ -265,3 +265,71 @@ export const mapCustomsDeclarations = ({
     mapCustomsDeclaration(declaration, notificationStatuses)
   )
 }
+
+const getBtmsDecision = (clearanceDecision) => {
+  for (const decisionCheck of btmsDecisionChecks) {
+    if (decisionCheck.checkType === 'item' && clearanceDecision.items.some(item => item.checks.some(itemCheck =>
+        itemCheck.decisionCode === decisionCheck.decisionCode && (decisionCheck.checkCode === undefined || itemCheck.checkCode === decisionCheck.checkCode)))) {
+      return decisionCheck.decision
+    }
+
+    if (decisionCheck.checkType === 'result' && clearanceDecision.results.some(result => result.internalDecisionCode === decisionCheck.decisionCode)) {
+      return decisionCheck.decision
+    }
+  }
+
+  return ''
+}
+
+// Order of these checks matter. It returns the 'worst' case of all the item decisions first.
+const btmsDecisionChecks = [
+  { checkType: 'item', decisionCode: 'E03', decision: 'Data Error - Unexpected data - transit, transhipment or specific warehouse' },
```

Comment:
rename `checkType` to `type`
rename `decisionCode to `code`
rename `decision` to `description`